saleor/plugins/webhook/tests/test_payment_webhook_utils.py

```diff
@@ -160,3 +163,25 @@ def test_parse_payment_action_response_parse_amount(
         dummy_webhook_app_payment_data, payment_action_response, TransactionKind.AUTH
     )
     assert gateway_response.amount == dummy_webhook_app_payment_data.amount
+
+
+def test_clear_successful_delivery(event_delivery):
+    # given
+    assert EventDelivery.objects.filter(pk=event_delivery.pk).exists()
```

Comment:
Do we really need assert a fixture?

Reply:
asserting fixture is not needed indeed. Although I'm not sure about fixtures since at the moment EventDelivery can have three different statuses and are created with "pending" as a default. It seems to me that status manipulating inside test is a bit less confusing and more explicit than adding another two fixtures for every status. Although I might be wrong and if you insist I will change it :)